docs/howto/hooks/webhooks.md

```diff
@@ -73,13 +73,15 @@ Upon execution, a webhook will send a request containing a JSON object with the
 | commit_message[^2]  | The message for the commit (or merge) that is taking place        | string |
 | committer[^2]       | Name of the committer                                             | string |
 | commit_metadata[^2] | The metadata for the commit that is taking place                  | string |
-| commit_id[^2,^4]    | The ID of the commit that is being created              | string |
+| commit_id[^2,^4]    | The ID of the commit that is being created                        | string |
```

Comment:
not rendering (see screenshot in comment blueish' arrow)

Reply:
Thanks, I'll fix it